src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mds-summary.pipe.ts

```diff
@@ -42,22 +42,49 @@ export class MdsSummaryPipe implements PipeTransform {
       contentLine1 = `${active} ${this.i18n('active')}`;
       contentLine2 = `${standbys + standbyReplay} ${this.i18n('standby')}`;
     }
-
+    let standbyHoverText = '';
+    let standbyflg = false;
+    for (const i in value.standbys) {
+      if (standbyflg) {
+        standbyHoverText = standbyHoverText + ', ' +  value.standbys[i].name;
+      } else {
+        standbyHoverText = standbyHoverText + value.standbys[i].name;
+        standbyflg = true;
+      }
+    }
+    const fsLength = (value.filesystems) ? value.filesystems.length : 0;
+    const infoObject = fsLength > 0 ? value.filesystems[0].mdsmap.info : {};
+    let activeHoverText;
+    Object.keys(infoObject).forEach(key => {
+      activeHoverText = infoObject[key].name;
+    });
+    const standbyTitleText = !standbyHoverText
+      ? ''
+      : `${this.i18n('standby daemons')}: ${standbyHoverText}`;
+    let activeTitleText = _.isUndefined(activeHoverText)
```

Comment:
For the above suggestion to work you have to remove the check for only `undefined` to also include other falsy states.
```suggestion
    let activeTitleText = !activeHoverText
```